### PR TITLE
Document our proof-of-concept

### PR DESCRIPTION
**Pull request by glauco**
This commit:
* Document the monitoring architecture
* Document how we are using: Logstash, Elasticsearch, Kibana and Filebeat
* Document how to use it on a new project

Pairing with @lucaslago

### CODE REVIEW

**Comment by lucaslago** [comment]
👍

File: src/app.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const log = bunyan.createLogger({
   name: 'fuse-equipment-api',
   streams: [{
     level: 'info',
-    path: 'log/equipment-api.log'
+    path: 'monitoring/log/equipment-api.log'
```
Comment (waldemarnt) [inline src/app.js]: Did you change the **filebeat.yml** to listen this path?

Comment (lucaslago) [inline src/app.js]: It wasn't needed, Filebeat is still listening to /var/log inside the container and we refactored our docker-compose.yml to  pass the correct folder as volume.

File: monitoring/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+## Requirements
+
+* [Virtualbox](https://www.virtualbox.org/)
+* [Docker](https://docs.docker.com/engine/installation)
+* [Docker Compose](https://docs.docker.com/compose/install)
+
+## Running it
+
+```js
```
Comment (jeffhsta) [inline monitoring/README.md]: it should be shell

**Comment by jeffhsta** [comment]
:+1:

**Pull request merged**